File: BookShop.DataAccess/Models/SearchParams.cs
using static BookShop.DataAccess.Entities.Enums.Enums.EntityFields;

namespace BookShop.DataAccess.Models
{
    public class SearchParams
    {
        public PrintedEditionType PrintedEditionType { get; set; }
        public SortCriteria SortCriteria { get; set; }
        public decimal PriceFrom { get; set; }
        public decimal PriceTo { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }

        public SearchParams()
        {
            PrintedEditionType = PrintedEditionType.None;
            SortCriteria = SortCriteria.None;
            PriceFrom = 0;
            PriceTo = decimal.MaxValue;
            Page = 1;
            PageSize = 6;
        }
    }
}

File: BookShop.ClientApp/src/app/models/searchParams.ts
import { SortCriteria } from 'src/app/enums/sortCriteria';
import { PrintedEditionType } from 'src/app/enums/printedEditionType';

export class SearchParams {
    public priceFrom: number;
    public priceTo: number;
    public sortCriteria: SortCriteria;
    public printedEditionType:PrintedEditionType;
    public page:number;
    public pageSize:number;
    public keyWord:string;

    constructor(
        ) { 
            this.keyWord='';
            this.priceFrom=0;
            this.priceTo=0;
            this.sortCriteria=SortCriteria.None;
        }
        
}